phase2.Rmd

```diff
@@ -121,8 +122,8 @@ grouped_review_stats = user_reviews_df %>%
   group_by(App) %>%
   summarize(
     n = n(), 
-    average_polarity = mean(Sentiment_Polarity),
-    average_subjectivity = mean(Sentiment_Subjectivity)
+    median_polarity = median(Sentiment_Polarity),
+    median_subjectivity = median(Sentiment_Subjectivity)
```

Comment:
@keyur2808 Could you explain the reasoning for this?

Reply:
Using median as it's less sensitve to outliers [ We don't want one person putting like -1 affecting others ]
grouped_review_stats = user_reviews_df %>%